modules/internal/DefaultSparse.chpl

```diff
@@ -269,6 +293,28 @@ module DefaultSparse {
       for e in data[1..dom.nnz] do yield e;
     }
 
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
```

Comment:
It took me a moment to realize David's right.  But this is a case like we were discussing in yesterday's call where we could factor all the common stuff (say, everything that computed 1..numChunks and/or startIx..endIx into a helper standalone iterator that both standalone iterators (and the leader iterators?) could all call into.

I vote for committing a first draft for all our main data structures before embarking on this refactoring, though.